in_toto/runlib.py

```diff
@@ -137,18 +138,29 @@ def record_artifacts_as_dict(artifacts):
     original_cwd = os.getcwd()
     try:
       os.chdir(in_toto.settings.ARTIFACT_BASE_PATH)
-    except OSError as e:
-      raise OSError("Review your ARTIFACT_BASE_PATH setting - {}".format(e))
+    except Exception as e:
+      raise in_toto.exceptions.SettingsError(
+          "Review your ARTIFACT_BASE_PATH setting '{}' - {}".format(
```

Comment:
Consider providing more information on what could have gone wrong.  For example, an invalid path.  On the other hand, appending the exception, as you have done, would make it clear what is wrong with `ARTIFACT_BASE_PATH`.

Reply:
I think adding the exception message is fine. I guess we can always enhance this later, e.g. when taking care of #117.